Formula/v/vim.rb

```diff
@@ -61,6 +61,10 @@ def install
     # vim doesn't require any Python package, unset PYTHONPATH.
     ENV.delete("PYTHONPATH")
 
+    if DevelopmentTools.clang_build_version == 1600
+      ENV.append_to_cflags "-mllvm -enable-constraint-elimination=0"
+    end
```

Comment:
```suggestion
    ENV.append_to_cflags "-mllvm -enable-constraint-elimination=0" if DevelopmentTools.clang_build_version == 1600
```
Should fix your syntax failure.